app/views/questions/show.html.erb

```diff
@@ -12,6 +12,9 @@
         <br>
         <div id="question-body" >
           <%= @question.body %><br>
+          <div id="delete-question">
+            <%= button_to "Delete Question", question_path(@question), method: :delete, class: "waves-effect red darken-1 waves-light btn right" %>
```

Comment:
Do we have a way of only showing the delete button when the question belongs to *current_user*. Otherwise whats stopping someone else from deleting your content? Maybe we could try a conditional that checks the user against the user_id on the question.